codex_of_the_damned/templates/layout.html

```diff
@@ -5,7 +5,7 @@
     {% block head %}
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta property="og:title" content="{{ og_title|default('Codex of the Damned') }}" />
+    <meta property="og:title" content="{{ title() }}" />
```

Comment:
You should not change this. This means that if your context returns `og_title`, it will replace the default title with it. The way you do it will preempt the og presentation that works for cards search.

Reply:
I commited a new way that solves this issue